roles/mysql/tasks/main.yml

```diff
@@ -55,3 +55,8 @@
     - database_clients
   tags: mysql
 
+- name: Create development database user
+  mysql_user: name={{ item.name}} host=% password={{ item.password }} priv=*.*:ALL state=present
+  with_items:
+    - "{{ databases.development_user }}"
+  when: develop
```

Comment:
We should document somewhere that the database username and password, when developing, is `development:development`.